pkg/cli/cli.go

```diff
@@ -462,3 +462,8 @@ func (c CLI) metadata() plugin.CLIMetadata {
 func (c CLI) Run() error {
 	return c.cmd.Execute()
 }
+
+// Command returns the underlying root command.
+func (c CLI) Command() *cobra.Command {
+	return c.cmd
+}
```

Comment:
Just to help us understand. 
How will you use it? What are you trying to do that is not possible without add this one?
Could you please provide an example?
(snippet code)